FSharpBuild.Directory.Build.props

```diff
@@ -42,7 +42,10 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(FSharpSourcesRoot)\fsharp\msft.pubkey</AssemblyOriginatorKeyFile>
     <StrongNames>true</StrongNames>
-    <DelaySign>true</DelaySign>
+    <DelaySign>false</DelaySign>
+    <PublicSign>true</PublicSign>
+    <GenerateSatelliteAssembliesForCore>true</GenerateSatelliteAssembliesForCore>
+    <NoCompilerStandardLib Condition="'$(TargetFramework)' == 'netstandard1.6'">true</NoCompilerStandardLib>
```

Comment:
Ditto?

Reply:
Some additional comments.
- What happens with ngen if run in an administrator window?
- Have you run F# in the ide with this change?
- Does FSI run as expected in an F5 environment?

Reply:
> What happens with ngen …

NGEN and shadow copying are two actions that trigger strong name validation. They will fail on public-sign assemblies, but they are also easily avoided actions.

> Have you run F# in the ide with this change?

No.

> Does FSI run as expected in an F5 environment?

I have not tested this.

💭 It's not clear whether disabling strong name validation applies to both public-sign and delay-sign assemblies. If it applies to both, then we could leave that portion in place for this change to have no impact on users who still choose to build with admin permissions.

Reply:
> Ditto?

I can't figure out where this gets set normally. I was expecting it to get set whenever `NoStdLib` is set, but for some reason it's not.